### ISSUE TITLE
email seems not to be sent on callback done?..

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/dispatcher-app#125]
need to inspect conditions and test better.

**Comment by burnout87** [comment]
Well, it should. There is also some test specific.

**Comment by volodymyrss** [comment]
Yeah, it's hard to diagnose. It's nice now I can see in the directory emails and job statuses.
In the future, we may think to make a call to inspect them all and validate the state.
For now, I added more log there #124 for better inspection.
